pragma/tests/fetchers_test.py

```diff
@@ -184,6 +184,79 @@
             ),
         ],
     },
+    "OkxFutureFetcher": {
+        "mock_file": MOCK_DIR / "responses" / "okx_future" / "ticker.json",
+        "fetcher_class": OkxFutureFetcher,
+        "name": "OKX",
+        "other_mock_fns": [
+            {
+                "format_expiry_timestamp_url": {
+                    "kwargs": {
+                        "BTC": {"id": "BTC-USD-230922"},
+                        "ETH": {"id": "ETH-USD-230922"},
+                    },
+                    "mock_file": MOCK_DIR
+                    / "responses"
+                    / "okx_future"
+                    / "timestamp_230922.json",
+                }
+            },
+            {
+                "format_expiry_timestamp_url": {
+                    "kwargs": {
+                        "BTC": {"id": "BTC-USD-230929"},
+                        "ETH": {"id": "ETH-USD-230929"},
+                    },
+                    "mock_file": MOCK_DIR
+                    / "responses"
+                    / "okx_future"
+                    / "timestamp_230929.json",
+                }
+            },
+        ],
+        "expected_result": [
+            FutureEntry(
+                pair_id="BTC/USD",
+                price=2664490000000,
+                timestamp=1695293953,
+                source="OKX",
+                publisher=PUBLISHER_NAME,
+                volume=73224927992200000700416,
+                expiry_timestamp=1695369600000,
+                autoscale_volume=False,
+            ),
+            FutureEntry(
+                pair_id="BTC/USD",
+                price=2666120000000,
+                timestamp=1695293953,
+                source="OKX",
+                publisher=PUBLISHER_NAME,
+                volume=271979672734799985901568,
+                expiry_timestamp=1695974400000,
+                autoscale_volume=False,
+            ),
+            FutureEntry(
+                pair_id="ETH/USD",
+                price=159390000000,
+                timestamp=1695293986,
+                source="OKX",
+                publisher=PUBLISHER_NAME,
+                volume=36278392896900000907264,
+                expiry_timestamp=1695369600000,
+                autoscale_volume=False,
+            ),
+            FutureEntry(
+                pair_id="ETH/USD",
+                price=159092000000,
+                timestamp=1695293987,
+                source="OKX",
+                publisher=PUBLISHER_NAME,
+                volume=98295299247559996866560,
+                expiry_timestamp=1695974400000,
+                autoscale_volume=False,
+            ),
+        ],
+    },
```

Comment:
could we move the configs to a separate file ?

Reply:
Yes. In addition, we can separate tests for regular `Fetcher` classes and `FutureFetcher`.

Reply:
Yes. In addition we can separate the tests for `Fetcher` and `FutureFetcher` classes.

Reply:
sounds good!